C/c4CAPI.cc

```diff
@@ -371,9 +372,11 @@ bool c4coll_purgeDoc(C4Collection *coll,
 {
     returnIfCollectionInvalid(coll, outError, false);
     try {
-        if (coll->purgeDocument(docID))
+        if (coll->purgeDocument(docID)) {
+            C4CollectionSpec spec = c4coll_getSpec(coll);
+            Log("API purge doc: %.*s.%.*s.%.*s", SPLAT(spec.scope), SPLAT(spec.name), SPLAT(docID));
```

Comment:
It's kind of weird to have "API" in the log message. Can we have something like "Purge doc : scope.collection.id" or Purge doc 'scope.collection.id' instead?

Reply:
I tried to assign reason of high level. "API Purge" means that it's directly called from Platform SDK. But I can do as you suggested. "API" kind of weired. We can always tell where it is called from reading the source.